Shared/Actions/Shots/HitscanShot.gd

```diff
@@ -43,15 +55,17 @@ func _physics_process(delta):
 	if  _first_frame:
 		_update_collision()
 		_first_frame = false
+	
+	_current_animated_range += _animated_range_increase_velocity * delta
+	_update_visual_range()
 
 
 func handle_hit(collider):
 	var character = collider.get_parent()
 	Logger.debug("hit collider: %s" %[collider.get_class()] , "HitscanShot")
 	var collision_point = get_collision_point()
 	var distance = (collision_point- global_transform.origin).length()
-	$Visualisation.scale.y = distance*0.5
-	$Visualisation.transform.origin.z = -distance*0.5
+	_current_range = distance
 
 	if _first_frame:
```

Comment:
We actually only update collisions during the first frame for some reason. ( I think this is also a relict from the prototype). I think we should change that so we don't have visual hits that don't trigger anything.

Reply:
Oh really? damn, yeah I'll change that